release-notes.md

```diff
@@ -1,5 +1,16 @@
 # Release notes
 
+## August 6, 2019
+
+The following updates were made in this release:
+
+**iOS Target 2.1.2 and Android Target 1.1.2**
+
+* `target.previewEnabled`, a new configuration Boolean key, has been added. This key is used to enable/disable Target Preview.
+  If key is not provided, preview is enabled by default.
+* Fixed an issue in Android where Target Preview links were decoded twice, which lead to an error preview page.
+* Fixed a bug that, when `prefetchContent` is called with a nil callback, caused a crash in iOS.
```

Comment:
This last bug fix didn't make it in the release -  `Fixed a bug that, when `prefetchContent` is called with a nil callback, caused a crash in iOS.` due to some git issue. We can omit it.

Reply:
sure, removed